examples/vm_scheduling_example/main.tf

```
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2021 Google LLC
```

nit: Should these (and other new files) be 2022?

```suggestion
 * Copyright 2022 Google LLC
```